### ISSUE TITLE
[Course List] Disclaimer lacks heading

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wai-evaluation-tools-list#60]
The disclaimer is just a `<header>` element without a heading. It is styled as a heading, and it should be a heading to improve navigation for screen reader users.

**Comment by mhansma96** [comment]
The header is now a H3 element this should help
